bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestBotFactory.java

```diff
@@ -158,6 +158,9 @@ public List<Bot> create(BotConfiguration configuration) {
             if (repo.value().contains("censuslink")) {
                 botBuilder.censusLink(repo.value().get("censuslink").asString());
             }
+            if (repo.value().contains("enable-csr")) {
```

Comment:
I think the option should just be "csr".

Reply:
Fixed.